evals/handlers/setup-context.ts

```diff
@@ -0,0 +1,114 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { createAdapters } from "../../src/adapters";
+import { CommentSimilaritySearchResult } from "../../src/adapters/supabase/helpers/comment";
+import { IssueSimilaritySearchResult } from "../../src/adapters/supabase/helpers/issues";
+import { fetchRepoLanguageStats, fetchRepoDependencies } from "../../src/handlers/ground-truths/chat-bot";
+import { findGroundTruths } from "../../src/handlers/ground-truths/find-ground-truths";
+import { logger } from "../../src/helpers/errors";
+import { formatChatHistory } from "../../src/helpers/format-chat-history";
+import { recursivelyFetchLinkedIssues } from "../../src/helpers/issue-fetching";
+import { Context } from "../../src/types";
+import { VoyageAIClient } from "voyageai";
+import OpenAI from "openai";
+
+const SEPERATOR = "######################################################\n";
+
+export interface FetchContext {
+  rerankedText: string[];
+  formattedChat: string[];
+  groundTruths: string[];
+}
+
+export interface EvalClients {
+  supabase: SupabaseClient;
+  voyage: VoyageAIClient;
+  openai: OpenAI;
+}
+
+export const initAdapters = (context: Context, clients: EvalClients): Context => {
```

Comment:
This shouldn't have passed linter settings. Make sure you run `yarn format` in case the git hooks aren't working. We don't generally allow arrow functions unless they are used in some tiny callback context.